docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,417 @@
+| Client refresh mode | Prebid.js automatically refreshes the tokens internally.<br/>This is the simplest approach. | [Client Refresh Mode](#client-refresh-mode) |
+| Server-only mode | Prebid.js doesn't automatically refresh the token. It is up to the publisher to manage token refresh.<br/>Choose this option if you want to use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md) to refresh the token, and Prebid.js to send the token to the bid stream. | [Server-Only Mode](#server-only-mode) |
+
+### Client Refresh Mode
+
+You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+
+For an example, see [Sample Token](#sample-token).
+
+As long as the refresh token remains valid, the UID2 Prebid module refreshes the UID2 token as needed.
+
+This section includes the following information:
+- [Client Refresh Mode Response Storage Options](#client-refresh-mode-response-storage-options)
+- [Client Refresh Mode Cookie Example](#client-refresh-mode-cookie-example)
+- [Client Refresh Mode uid2Token Example](#client-refresh-mode-uid2token-example)
+- [Passing a New Token: Client Refresh Mode](#passing-a-new-token-client-refresh-mode)
+
+#### Client Refresh Mode Response Storage Options
+
+When you configure the module to use Client Refresh mode, you must choose **one** of the following options for storing the API response information.
+
+| Option | Details | Use Case | 
+| --- | --- | --- |
+| Set `params.uid2Cookie` to the name of the cookie that contains the response body as a JSON string. | See [Client Refresh Cookie Example](#client-refresh-cookie-example). | Use this option only when there is enough space left in your cookie to store the response body. (**GWH_LP do we need to address how user will know if there is space in the cookie?**)|
+| Set `params.uid2Token` to the response body as a JavaScript object. | See [Client Refresh uid2Token Example](#client-refresh-uid2token-example). | You might choose to provide the response body via `params.uid2Token` in either of these cases:<ul><li>If storing the response body on the cookie will exceed the cookie size limit.</li><li>If you prefer to have the flexibility to manage the storage of the response body yourself.</li></ul> |
+
+#### Client Refresh Mode Cookie Example
+
+In Client Refresh mode, Prebid.js takes care of refreshing the token. To do this, you must configure it to store the token. The following example shows the cookie and also the configuration for storing the token in a cookie called `uid2_pub_cookie`.
+
+Cookie:
+
+```
+uid2_pub_cookie={"advertising_token":"...advertising token...","refresh_token":"...refresh token...","identity_expires":1684741472161,"refresh_from":1684741425653,"refresh_expires":1684784643668,"refresh_response_key":"...response key..."}
+```
```

Comment:
I think we can link to _Sample Token_ section for a second clearer example. We still want the code example above because we want to show pub that they need to set the value to uid2_pub_cookie object.

Reply:
@sunnywu done but not 100% tracking -- please check I addressed this correctly. Currently it's line 163.

Reply:
looks ok to me